pkg/kubefed2/enable/enable.go

```diff
@@ -255,6 +257,7 @@ func CreateResources(cmdOut io.Writer, config *rest.Config, resources *typeResou
 		if err != nil {
 			return errors.Wrapf(err, "Error creating CRD %q", resources.CRD.Name)
 		}
+
```

Comment:
Please remove the inserted newlines - unrelated and nonfunctional formatting changes do not belong in this PR.

Reply:
removed